atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/config/TargetedSweepRuntimeConfig.java

```diff
@@ -51,6 +51,11 @@ void checkShardSize() {
                 "Shard number must be between 1 and 256 inclusive, but it is %s.", shards());
     }
 
+    @Value.Default
+    public long pauseMillis() {
+        return 500L;
```

Comment:
do you want to keep the default the same, adjust on particular deployment and change default if it's not too aggressive?

Reply:
not too fussed either way

Reply:
I think the default is too high for no real reason